src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/UseGenericEventHandlerInstances.cs

```diff
@@ -95,39 +94,14 @@ public override void Initialize(AnalysisContext analysisContext)
                     static bool IsDelegateTypeWithInvokeMethod(INamedTypeSymbol namedType) =>
                         namedType.TypeKind == TypeKind.Delegate && namedType.DelegateInvokeMethod != null;
 
-                    bool IsEventArgsParameter(IParameterSymbol parameter)
-                    {
-                        var type = parameter.Type;
-                        if (IsAssignableTo(type, eventArgs, context.Compilation))
```

Comment:
Can we delete this protected abstract method now?

Reply:
Why not?

Reply:
I meant can you now delete `IsAssignableTo`?

Reply:
That's why I love a good review!